dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -750,6 +752,15 @@
 
     }
 
+    function notifyUVEWhenURLisChanged(data) {
+        var customEvent = document.createEvent('CustomEvent');
+        customEvent.initCustomEvent('ng-event', false, false, {
+            name: 'url-is-changed',
+            payload: data
+        });
+        document.dispatchEvent(customEvent);
+    }
```

Comment:
We have the `save/save-page` event in this file. We can include the new `URL` as part of that payload.

Reply:
The event save/save-page is not triggered when change a property in the same "flow" event as create a language

Reply:
We don't care about the changing unless we save/publish, at least for now. 

We should check this on save and I believe we are getting the URL on the payload of that event

Reply:
Check [my comment](https://github.com/dotCMS/core/pull/30697#issuecomment-2486712535) below